tests/integration/transformers/test_categorical.py

```diff
@@ -47,7 +47,7 @@ def test_clip_interval(self):
     def test__reverse_transform(self):
         """Test the ``reverse_transform``."""
         # Setup
-        data = pd.DataFrame({'column_name': [1, 2, 3, 2, 2, 1, 3, 3, 2]})
+        data = pd.DataFrame({'column_name': [-1, 0, 1, 2, 3, 2, 2, 1, 3, 3, 2, -11, 0]})
```

Comment:
It'll be easier to test in unit test `test__reverse_transform`.  We'll want to modify the `transformed` data variable, not the original data